src/main/resources/application.yml

```diff
@@ -60,6 +60,17 @@ sandbox:
   validation:
     executor:
       pool-size: 10
+    predefinedSchemas:
+      EDM-INTERNAL:
+        url:
+        rootLocation:
+        schematronLocation:
+      EDM-EXTERNAL:
+        url:
+        rootLocation:
+        schematronLocation:
+  transformation:
+    xslt-url:
```

Comment:
url goes here, it should be the url to the file which can be local or remote
Example: `file:/Users/user/documents/path/to/xstl/default.xsl`